Actions/AnalyzeTests/AnalyzeTests.ps1

```diff
@@ -17,25 +17,41 @@ try {
     . (Join-Path -Path $PSScriptRoot 'TestResultAnalyzer.ps1')
 
     $testResultsFile = Join-Path $ENV:GITHUB_WORKSPACE "$project\TestResults.xml"
-    if (Test-Path $testResultsFile) {
-        $testResults = [xml](Get-Content "$project\TestResults.xml")
-        $testResultSummary = GetTestResultSummary -testResults $testResults -includeFailures 50
+    $testResultsSummaryMD, $testResultsfailuresMD, $testResultsFailuresSummaryMD = GetTestResultSummaryMD -path $testResultsFile
 
-        Add-Content -Encoding UTF8 -Path $env:GITHUB_OUTPUT -Value "TestResultMD=$testResultSummary"
-        Write-Host "TestResultMD=$testResultSummary"
+    $settings = $env:Settings | ConvertFrom-Json
+    Write-Host "thresholds type: $($settings.bcptThresholds.GetType().FullName)"
+    $bcptTestResultsFile = Join-Path $ENV:GITHUB_WORKSPACE "$project\bcptTestResults.json"
+    $bcptBaseLineFile = Join-Path $ENV:GITHUB_WORKSPACE "$project\bcptBaseLine.json"
+    $bcptThresholdsFile = Join-Path $ENV:GITHUB_WORKSPACE "$project\bcptThresholds.json"
```

Comment:
The file is really there to be able to modify the threshold for individual projects easily.
I was debating with myself whether the setting should be there at all.

Reply:
If each individual project already has an AL-Go settings file, wouldn't it be just as easy to change the thresholds in there?

Reply:
I am just thinking about how the partner is working - IMO settings shouldn't be for temporary things.
Disabling tests are also not done in settings - that is also done in a file.

Reply:
The idea is that if I need to modify thresholds temporarily - I can do this in a file and simply delete the file later.
If I modify settings globally or project wise - I would need to remember or look in git history to figure out what the old threshold was.
Same with disabledtests.json - you can temporarily disable tests using this file - instead of removing test attributes from test apps or like.

Reply:
> I would need to remember or look in git history to figure out what the old threshold was.

Luckily, it's only a few clicks in the GitHub UI 😉